src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display({ result }) {
+  return <h1>{result}</h1>;
+}
+
+Display.propTypes = {
+  result: PropTypes.string,
+};
+
+Display.defaultProps = {
+  result: 0,
+};
```

Comment:
given that it's string, we should have 
```
  result: '0',
```

Reply:
Fixed them both. 

Thank you for trusting me. 

Nick